### DISCUSSION TITLE
Pre-Analysis

### DISCUSSION THREAD

**Discussion by leporia** [prosyslab-classroom/cs524-program-analysis#429] [Art competition]
![_cebf2b95-964b-4581-84ec-fc9db12db134](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/20344547/a5c46f30-f72d-44f9-9e90-ee23bb8bdbee)

## Name
Andrea Lepori

## Tool You Used
[Bing Image Creator](https://www.bing.com/create)

## Short Description
For me, Pre-Analysis means "a split frame on one side a sketch of a half done crane and on the other side a colored photo realistic big building".

## Long Description (up to 3 sentences)
The pre-analysis builds a sketch (left side) of the reality to then aid the final analysis (right side) to construct a more realistic approximation. Even though the final analysis is more precise it cannot fully capture everything that is why we still have a white border around it. We can further see that going from left to right it gets more detailed in the same way that with more iterations/time we get a more precise result.

**Comment by KihongHeo** [comment]
Very good. Make sense.
